tests/test_reader.py

```diff
@@ -35,6 +35,19 @@ def test_tfs_read_str_input(self, _tfs_file_str: str):
         assert len(str(test_file)) > 0
         assert isinstance(test_file.index[0], str)
 
+    def test_tfs_read_no_validation(self, _tfs_file_pathlib: pathlib.Path):
+        test_file = read_tfs(_tfs_file_pathlib, index="NAME", validate_after_reading=False)
+        assert len(test_file.headers) > 0
+        assert len(test_file.columns) > 0
+        assert len(test_file.index) > 0
+        assert len(str(test_file)) > 0
+        assert isinstance(test_file.index[0], str)
```

Comment:
I would also test a file, that would fail validation with an error. (not only the warning, below)